app/controllers/posts_controller.rb

```diff
@@ -27,6 +27,17 @@ def create
     end
   end
 
```

Comment:
- [ ] Please make sure that you use `CanCanCan` for Post authorization in the Post Controller to automatically authorize all actions. :relaxed: 
  ```rb
    class PostsController < ApplicationController
      load_and_authorize_resource
  
      ...
    end
  ```